.github/workflows/test.yml

```diff
@@ -170,6 +170,81 @@ jobs:
           name: coverage-integration-test-${{ matrix.python-version }}-${{ matrix.airflow-version }}
           path: .coverage
 
+  Run-Integration-Tests-Remote-Cache-Path:
```

Comment:
@pankajkoti why did we create a dedicated job for these tests?

Reply:
The current integration tests have turned off dbt ls cache. I was unsure on why they are turned off and additionally kept a separate job for remote cache (in case we would have liked the existing job to follow the local/default/variable based cache)

Reply:
Removed the integration tests as part of this PR and will create a follow-up PR to have limited tests running as discussed.